x-pack/plugins/security_solution/cypress/tasks/risk_scores/transforms.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TRANSFORMS_URL } from '../../urls/risk_score';
+
+export const getTransformState = (transformId: string) => {
```

Comment:
Please, consider moving all the API calls to the `api_calls` folder inside the tasks one.